bedrock/firefox/templates/firefox/os/index-new.html

```diff
@@ -36,14 +50,18 @@
       <div class="content-wrapper">
         <div class="phone">
           <div class="fxos-cta">
-            <h1><span>{{_('Firefox OS')}}</span></h1>
+            <h1>{{ high_res_img('img/firefox/os-new/fxos-wordmark.png', {'alt': _('Firefox OS'), 'width': '420', 'height': '85'}) }}</h1>
```

Comment:
I'm going to leave this as-is for now. There are various techniques we could use for the fallback, but each comes at a cost. We could always file a follow-up bug to research this a bit more, but for now I don't think it's worth holding things up.